### ISSUE TITLE
Minifier cause JS error in generated HTML report

### ISSUE THREAD

**Issue by mr-beerkiss** [nearform/node-clinic-doctor#331]
Hello 👋

I came across Clinic/Doctor from [this article](https://www.smashingmagazine.com/2018/06/nodejs-tools-techniques-performance-servers/)

However, when I run `clinic doctor ... ` the generated HTML has styling but I couldn't see any of the juicy graphs and data I was expecting. When I opened the Dev Tools, the JS console shows the following error:

```
39771.clinic-doctor.html:1701 Uncaught SyntaxError: Missing initializer in const declaration
```

I then tried with [node-clinic-doctor-examples](https://github.com/nearform/node-clinic-doctor-examples) and had the same issue there. I decided to fork this repo so I could disable the [minifier](https://github.com/nearform/node-clinic-doctor/blob/master/index.js#L230) to pinpoint the JS issue and, lo and behold, without minification, it works!

For now, I can sidestep the issue by using the `--debug` flag. Hopefully, that will work for others too.

Running MacBook Pro (15-inch, 2018), macOS Catalina Version 10.15.5 (19F101)
Issue occurred on all the browsers I tried with:

  - Chrome Version 85.0.4158.4 (Official Build) dev (64-bit)
  - Vivaldi | 3.0.1874.38 (Stable channel) (64-bit)
  - Safari Version 13.1.1 (15609.2.9.1.2)

**Comment by jasnell** [comment]
yep, known issue. Hoping to have a fix out later today

**Comment by jasnell** [comment]
Fixed! The latest clinic 6.0.2 should work :-)
